School.js

```diff
@@ -12,9 +14,9 @@ class School {
   /**
    * Add class to classes
    * 
-   * @param {string} name - Name of the class
-   * @param {string} teacher - Name of instructor 
-   * @return {Class} Class object
+  //  * @param {string} name - Name of the class
+  //  * @param {string} teacher - Name of instructor 
+  //  * @return {Class} Class object
```

Comment:
You need to return in the function below. You should return `newClass` so later you can use that as a part of your response in your app.js file.